packages/account-v2/src/utils/accountLimitsUtils.ts

```diff
@@ -0,0 +1,140 @@
+import { GetLimits } from '@deriv/api-types';
+import { FormatUtils } from '@deriv-com/utils';
+import { TCurrency } from '../types';
+
+export const CATEGORY = {
+    footer: 'footer',
+    header: 'header',
+    submarket: 'submarket',
+};
```

Comment:
```suggestion
} as const;
```
should we add it? if it is a config